editor/src/clj/editor/game_object.clj

```diff
@@ -140,6 +140,13 @@
       (update :display-order (fn [display-order]
                                (vec (distinct (concat display-order (:display-order source-properties))))))))
 
+(defn resource-path-error [_node-id source-resource]
+  (let [node-type (some-> source-resource resource/resource-type :node-type)]
+    (or (validation/prop-error :fatal _node-id :path validation/prop-nil? source-resource "Path")
```

Comment:
`:fatal` fits better in this case than `:info` (if user ignores this info and build - it will be an exception in another place).